src/main/java/net/dancier/dancer/chat/dto/MessageDto.java

```diff
@@ -0,0 +1,20 @@
+package net.dancier.dancer.chat.dto;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Data;
+
+import java.time.OffsetDateTime;
+import java.util.Set;
+import java.util.UUID;
+
+@Data
+public class MessageDto {
+    private UUID id;
+    private UUID author;
+    private String text;
+    private Set<UUID> readByUsers;
```

Comment:
Yes, sounds good.
The notifiied dancer is I think not at all needed in the frontend. Would be only interesting in the backend to decide when to again notify the user. I will remove it from here.

And we have no endpoint at the moment to update the read_by_dancer property. I would suggest, when someone has a chat open with new messages we make a BE call with the last read message id or timestamp and then update it in the chat dancer. But I am unsure at the moment what endpoint to use. It could be a PUT on the chat/id endpoint. But it feels not 100% right as it is not a property of a chat. We could also make a PUT on the last read message /chats/id/messages/id, but this would have site effects as it would set the read by users also on older messages which are now also read.